### PR TITLE
[Import features] Wait for inserts to finish before returning

### PR DESCRIPTION
**Pull request by gino-m**
@DaoyuT I ended up doing this because it was blocking me from being able to import GeoJSON so that I could test a fix on the Android client.

I also added messages to the response in `{error: '...'}` and count int `{count: ..}` while I was there.

May address #564 and #694. Contributes to #696. 

PTAL?

### CODE REVIEW

File: functions/import-csv.js [hunk 1]
```diff
@@ -64,19 +62,23 @@ async function importCsv(req, res) {
 
     file.pipe(csvParser()).on("data", async (row) => {
       try {
-        await insertRow(projectId, layerId, row);
+        inserts.push(insertRow(projectId, layerId, row));
       } catch (err) {
         console.error(err);
-        await res.status(HttpStatus.BAD_REQUEST).end("{}");
-        // TODO(#525): Abort stream on error. How?
+        re.unpipe(busboy);
```
Comment (DaoyuT) [inline functions/import-csv.js:68]: Is this supposed to be `res.unpipe()` ? Same question for below

**Review by DaoyuT — DISMISSED** [review]
LGTM

File: functions/import-csv.js (hunk 1 above)
Comment (gino-m) [reply functions/import-csv.js]: Yikes, last minute typo! Fixed.

> Same question for below

What were you referring to? Don't see another instance of `re.`..

Comment (DaoyuT) [reply functions/import-csv.js]: Oh maybe I was looking at the same one and did not notice, LGTM

**Review by DaoyuT — APPROVED** [review]

File: functions/import-csv.js (hunk 1 above)
Comment (gino-m) [reply functions/import-csv.js]: Thanks for reviewing!

**Pull request merged**